python/ql/test/query-tests/Security/CWE-022-PathInjection/path_injection.py

```diff
@@ -127,3 +127,20 @@ def stackoverflow_solution():
     if os.path.commonprefix((os.path.realpath(path), STATIC_DIR)) != STATIC_DIR:
         return "not this time"
     f = open(path) # OK TODO: FP
+
+
+@app.route("/getctime")
+def camera_img_acquire():
+    filename = request.args.get('filename', '')
+    image_path = os.path.join(STATIC_DIR, filename)
+    timestamp = os.path.getctime(image_path) # OK as exposure of ctime of a particular file does not pose a security thread
```

Comment:
Being able to determine file existence is an information leak, that is a known vulnerability. First results from a google search was https://www.acunetix.com/vulnerabilities/web/arbitrary-file-existence-disclosure-in-action-pack/ -- please change this test to say `NOT OK`